### PR TITLE
Fixed ewsghana tests

### PR DESCRIPTION
**Pull request by kkrampa**

### CODE REVIEW

**Comment by kkrampa** [comment]
https://github.com/dimagi/commcare-hq/pull/9906

File: custom/ewsghana/tests/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+    rsp2 = Location(
+        name='Test Regional Medical Store',
+        site_code='rsp2',
+        location_type='Regional Medical Store',
+        domain=domain,
+        parent=test_region2
+    )
+    rsp2.save()
+    assign_products_to_locations(rsp, ["ad", "al"])
```
Comment (gcapalbo) [inline custom/ewsghana/tests/test_utils.py]: should this be `rsp2`?

**Comment by gcapalbo** [comment]
@kkrampa one comment otherwise looks good

**Comment by kkrampa** [comment]
@gcapalbo Fixed.

**Comment by gcapalbo** [comment]
strange, i'm not sure why a build didn't start on the latest commit, but the only test affected by that typo was the `TestInputStockView` test and it passed locally for me, so going to merge

**Comment by gcapalbo** [comment]
(I didn't realize this was merging into branch sync-sync-supply-points - @esoergel fyi you'll want to update your local branch. Kamil fixed some ews tests pertaining to your changes on that branch)

**Comment by kkrampa** [comment]
@gcapalbo I made this PR in response to this case: http://manage.dimagi.com/default.asp?191032.
I forgot to paste link to case.

**Comment by gcapalbo** [comment]
@kkrampa yep, no problem, just wanted to let Ethan know to update his local branch

**Comment by esoergel** [comment]
Thanks for the quick response, @kkrampa .  And thanks for the heads-up, @gcapalbo

**Pull request merged**